lib/VariableResolver.js

```diff
@@ -0,0 +1,246 @@
+import { getProcessVariables } from '@bpmn-io/extract-process-variables/zeebe';
+import { getBusinessObject, is } from 'bpmn-js/lib/util/ModelUtil';
+import CachedValue from './util/CachedValue';
+import { hasInputParameter, hasOutputMappings } from './util/ExtensionElementsUtil';
+
+export class VariableResolver {
```

Comment:
As I understand this is the main utility to retrieve variables (in a particular context)?

Let's add a little bit of additional documentation here as well as a meaningful README.